autofs-5.1.1 - revert fix libtirpc name clash

From: Ian Kent <ikent@redhat.com>

Commit 09d4edb2 introduced a substitution maro for the tirpc macro
auth_destroy() because it contained an unconditional call to a function
log_debug() which clashed with an autofs function of the same name.

AFAICS that was never in a actual libtirpc release, the call in a release
was auth_log_debug() which doesn't clash with any autofs functions.

So revert this change.

Signed-off-by: Ian Kent <ikent@redhat.com>
---
 CHANGELOG      |    1 +
 lib/rpc_subs.c |   10 ----------
 2 files changed, 1 insertion(+), 10 deletions(-)

diff --git a/CHANGELOG b/CHANGELOG
index 688d485..3a9266d 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,6 +1,7 @@
 ??/??/2015 autofs-5.1.2
 =======================
 - update libtirpc workaround for new soname.
+- revert fix libtirpc name clash.
 
 21/04/2015 autofs-5.1.1
 =======================
diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index 84fae9e..68d68f4 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -34,16 +34,6 @@
 #include <poll.h>
 
 #ifdef WITH_LIBTIRPC
-#undef auth_destroy
-#define auth_destroy(auth)                                              \
-                do {                                                    \
-                        int refs;                                       \
-                        if ((refs = auth_put((auth))) == 0)             \
-                                ((*((auth)->ah_ops->ah_destroy))(auth));\
-                } while (0)
-#endif
-
-#ifdef WITH_LIBTIRPC
 const rpcprog_t rpcb_prog = RPCBPROG;
 const rpcvers_t rpcb_version = RPCBVERS;
 #else
